app/site/_includes/project-type-tag.liquid

```diff
@@ -0,0 +1,14 @@
+{% case type %}
+  {% when "midsize" %}
```

Comment:
nit: for consistency, could you make "midsize" all lowercase in the metrics data for the field `nadia_badge_name`?